contrib/common.h

```diff
@@ -9,7 +9,9 @@
 // constants needed within common
 #define MAX_NQUAD 100              // max number of positive quadr nodes
 // increase this if you need >1TB RAM...
-#define MAX_NF    (BIGINT)1e11     // too big to ever succeed (next235 takes 1s)
+#define MAX_NF    (BIGINT)INT_MAX  // too big to ever succeed (next235 takes 1s)
```

Comment:
Please remove my comment, and replace with something relevant, eg: in this GPU version we only allow array sizes up to 2^31 which about 2e9.

Reply:
dont' worry, just did it -A